src/main/java/ua/ithillel/lesson15/Product.java

```diff
@@ -0,0 +1,61 @@
+package ua.ithillel.lesson15;
+
+import java.time.LocalDate;
+
+public class Product {
+    private final String name;
+    private final Category category;
+    private double price;
+    private final boolean discount;
+    private final LocalDate addingDate;
```

Comment:
```suggestion
    private final LocalDate additionDate;
```
часто будут зустрічатися (особливо з базами даних):  
creationDate, updateDate
або 
createdAt, updatedAt 
.